sdk/appcomplianceautomation/ci.yml

```diff
@@ -0,0 +1,32 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
This yaml is out of date, please rename it to `ci.mgmt.yml` and take this [file](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/apimanagement/ci.mgmt.yml) as an example to update the content.

Reply:
Fixed~Thanks!